bciers/apps/administration/app/components/operations/Operations.tsx

```diff
@@ -1,14 +1,53 @@
 import OperationDataGrid from "./OperationDataGrid";
 import { OperationRow, OperationsSearchParams } from "./types";
+import { Suspense } from "react";
+import Loading from "@bciers/components/loading/SkeletonGrid";
 import fetchOperationsPageData from "./fetchOperationsPageData";
+import Note from "@bciers/components/layout/Note";
+import Link from "next/link";
+import { Button } from "@mui/material";
+
+export const InternalUserOperationsLayout = ({
```

Comment:
Should these layouts live in their own file?

Reply:
@marcellmueller  I agree, and would vote yes to "their own file"...potentially beneficially for readability via folder structure